src/views/GameTemplates/GameTemplates.scss

```diff
@@ -0,0 +1,75 @@
+@import "../../index";
+
+.template-header {
+    display: flex;
+    color: $color-white;
+    font-size: 2rem;
+    justify-content: space-evenly;
+
+    .input-field {
+        display: flex;
+        flex-direction: row;
+        justify-content: space-evenly;
+        padding: .5em;
+
+        input {
+            height: 50px;
+            width: 300px;
+            font-size: 1rem;
+            border: round;
+            border-color: white;
+            // border-radius: 10px;
+            box-shadow: 6px 6px 14px -5px rgba(0, 0, 0, 0.5);
+            background-color: transparent;
+            color: white;
+        }
+    }
+}
+
+.my-server1 {
+    display: flex;
+    align-items: center;
+
+    justify-content: space-between;
+
+    border-bottom: 1px solid #000;
+    padding: 10px 30px;
+    color: $color-white;
+
+    .server-info {
+        h1 { font-size: 120%; }
+        h2 { font-size: 110%; }
+    }
+}
```

Comment:
After changing `my-server1`, `my-server2`, and `my-server3` classes to `template-list item`, we can condense these three style blocks into a single block.
```
.template-list-item {
    display: flex;
    align-items: center;
    justify-content: space-between;

    border-bottom: 1px solid #242424;
    padding: 10px 30px;
    color: $color-white;

    .server-info {
        h1 { font-size: 120%; }
        h2 { font-size: 110%; }
    }
}
```

This will apply the same style to all the list items without having to make a new style block for each one.